crates/op-rbuilder/src/integration/mod.rs

```diff
@@ -0,0 +1,182 @@
+use std::future::Future;
+use std::path::Path;
+use std::{
+    fs::{File, OpenOptions},
+    io,
+    io::prelude::*,
+    path::PathBuf,
+    process::{Child, Command},
+    time::{Duration, SystemTime},
+};
+use time::{format_description, OffsetDateTime};
+use tokio::time::sleep;
+
+/// Default JWT token for testing purposes
+pub const DEFAULT_JWT_TOKEN: &str =
```

Comment:
can move to a common crate for this constant

Reply:
Why do you feel we need that? The constant is only used in the context of that integration package. Note that eventually the integration folder will be a crate in itself to be shared with L1.